themes/digital.gov/layouts/shortcodes/note.html

```diff
@@ -1,129 +1,45 @@
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use
-          xlink:href="{{ "uswds/img/sprite.svg#assessment" | relURL }}"
-        ></use>
-      </svg>
-      Activity
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "action") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#campaign" | relURL }}"></use>
-      </svg>
-      Action
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "alert") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#report" | relURL }}"></use>
-      </svg>
-      Alert
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "comment") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#comment" | relURL }}"></use>
-      </svg>
-      Comment
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "video") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use
-          xlink:href="{{ "uswds/img/sprite.svg#fast_forward" | relURL }}"
-        ></use>
-      </svg>
-      Video
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "group") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#groups" | relURL }}"></use>
-      </svg>
-      Who can join?
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "plain") -}}
-  {{- else -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use
-          xlink:href="{{ "uswds/img/sprite.svg#notifications" | relURL }}"
-        ></use>
-      </svg>
-      Note
-    </h4>
-  {{- end -}}
-
-  {{- .Inner | markdownify -}}
-</div>
+{{ $variant := (.Get "variant") }}
+{{ $icon := "" }}
+
+{{ if eq $variant "Activity" }}
+  {{ $icon = "assessment" }}
+{{ else if eq $variant "Action" }}
+  {{ $icon = "campaign" }}
+{{ else if eq $variant "Alert" }}
+  {{ $icon = "report" }}
+{{ else if eq $variant "Comment" }}
+  {{ $icon = "comment" }}
+{{ else if eq $variant "Video" }}
+  {{ $icon = "fast_forward" }}
+{{ else if eq $variant "Who can Join?" }}
+  {{ $icon = "groups" }}
+{{ else if eq $variant "Note" }}
+  {{ $icon = "notifications" }}
+{{ else if eq $variant "Disclaimer" }}
+  {{ $icon = "info" }}
+{{ end }}
```

Comment:
We should check when no `$variant` is passed in the first check.
Can we just use `note` as the default when none is provided?

```
{{/*  set default when no variant is provided  */}}
{{ if eq $variant nil }}
  {{ $variant = "note" }}
  {{ $icon = "notifications" }}
{{ else if eq $variant "activity" }}
  {{ $icon = "assessment" }}
{{ else if eq $variant "action" }}
  {{ $icon = "campaign" }}
{{ else if eq $variant "alert" }}
  {{ $icon = "report" }}
{{ else if eq $variant "comment" }}
  {{ $icon = "comment" }}
{{ else if eq $variant "video" }}
  {{ $icon = "fast_forward" }}
{{ else if eq $variant "join" }}
  {{ $icon = "groups" }}
{{ else if eq $variant "note" }}
  {{ $icon = "notifications" }}
{{ else if eq $variant "disclaimer" }}
  {{ $icon = "info" }}
{{ end }}
```

Reply:
added condition for if someone does not add a variant then will default to the note format